server/src/models/Question.ts

```diff
@@ -0,0 +1,12 @@
+import mongoose, {Schema} from 'mongoose';
+let Schema = mongoose.Schema
+
+// TODO move those enum values to constants somewhere
+let QuestionSchema : Schema = new Schema({
+  prompt: {type: String, required: true},
+  options: {type: [String]},
+  type: {type: String, enum: ['CHOICE', 'TEXT'], required: true}
```

Comment:
The enum needs to be a list of all of the possible question types listed in Part A.
Nit: could move the enum to a const outside of the Schema